src/test/java/core/basesyntax/service/FruitTransactionsServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionsServiceImplTest {
+    private static FruitTransactionsService fruitTransactionsService;
+    private static List<FruitTransaction> fruitTransactions;
+    private static List<String> emptyList;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitTransactionsService = new FruitTransactionsServiceImpl();
+        initializeFruitTransactions();
+        emptyList = new ArrayList<>();
+    }
```

Comment:
```suggestion
    private static List<String> emptyList;
    private List<FruitTransaction> fruitTransactions;
    
    @BeforeClass
    public static void beforeClass() {
        fruitTransactionsService = new FruitTransactionsServiceImpl();
        emptyList = new ArrayList<>();
    }
    
    @Before
    public void init() {
        initializeFruitTransactions();
    }
```

It is better to initialize fruitTransactions before every test, so as not to transfer the state of fruitTransactions list from one test to another (same in other such places)